### PR TITLE
#120

### PR DESCRIPTION
**Pull request by akaric03**
http://cqmantis.dcouv.fr/view.php?id=120
Kongregate_red.png was left because I don't know what it does. Can be deleted in future update when we find out that the png. is not needed.

### CODE REVIEW

**Comment by Simoris** [comment]
Have you checked how the import battle-scrren looks (if the background- and window-color look good together)?

Kongregate_red.png is shown on the loading screen (only in kong version, not in the demo).

File: client/demo.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
             top: 195px;
             z-index: 1000;
             text-align: center;
-            background: url("img/WINDOW.png");
+            background: #e5bd6c;
```
Comment (MatthieuBonne) [inline client/demo.html:47]: i suggest #dddddd for more contrast

File: client/index.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
             top: 195px;
             z-index: 1000;
             text-align: center;
-            background: url("img/WINDOW.png");
+            background: #e5bd6c;
```
Comment (MatthieuBonne) [inline client/index.html:47]: same as above

**Review by MatthieuBonne — COMMENTED** [review]
better popup contrast and it'll be ok for me

File: client/demo.html (hunk 1 above)
Comment (MatthieuBonne) [inline client/demo.html:47]:
```suggestion
            background: #dddddd;
```

File: client/index.html (hunk 2 above)
Comment (MatthieuBonne) [reply client/index.html:47]:
```suggestion
            background: #dddddd;
```

**Pull request merged**